### PR TITLE
Add docs to support bring your own logs shipper

### PR DESCRIPTION
**Pull request by akodali18**
co-authored by : John Cornish (jcornish@vmware.com)

### CODE REVIEW

File: deploy/kubernetes/scenarios/wavefront-bring-your-own-logs-shipper.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  namespace: observability-system
+spec:
+  clusterName: YOUR_CLUSTER_NAME
+  wavefrontUrl: YOUR_WAVEFRONT_URL
+  dataCollection:
+    metrics:
+      enable: true
+  dataExport:
+    # Once deployed, the svc is available to consume logs at
+    # For json_array format: http://wavefront-proxy.observability-system.svc.cluster.local:2878/logs/json_array?f=logs_json_arr
```
Comment (mmichael) [inline deploy/kubernetes/scenarios/wavefront-bring-your-own-logs-shipper.yaml:15]: I'd suggest not putting the the namespace for the proxy service url.  You can just put in `http://wavefront-proxy:2878/logs/json_array?f=logs_json_arr` instead and you don't have to worry about if they change the operator / proxy namespace.   That is how we previously configured fluentd.   See https://github.com/wavefrontHQ/wavefront-operator-for-kubernetes/blob/v2.1.0/deploy/internal/logging/0-fluent-configmap.yaml#L113

File: deploy/internal/proxy/1-wavefront-proxy-deployment.yaml [hunk 2]
```diff
@@ -79,7 +79,6 @@ spec:
               {{- if .DataExport.WavefrontProxy.HttpProxy.HttpProxyUser }} --proxyUser {{ .DataExport.WavefrontProxy.HttpProxy.HttpProxyUser }} {{- end -}}
               {{- if .DataExport.WavefrontProxy.HttpProxy.HttpProxyPassword }} --proxyPassword {{ .DataExport.WavefrontProxy.HttpProxy.HttpProxyPassword }} {{- end -}}
               {{- if .DataCollection.Logging.Enable }} --customMessageTags log {{- end }}
-
```
Comment (mmichael) [inline deploy/internal/proxy/1-wavefront-proxy-deployment.yaml:82]: Is this change necessary?

File: deploy/kubernetes/scenarios/wavefront-bring-your-own-logs-shipper.yaml (hunk 1 above)
Comment (akodali18) [reply deploy/kubernetes/scenarios/wavefront-bring-your-own-logs-shipper.yaml:15]: Agree

File: deploy/internal/proxy/1-wavefront-proxy-deployment.yaml (hunk 2 above)
Comment (akodali18) [reply deploy/internal/proxy/1-wavefront-proxy-deployment.yaml:82]: fixed.

**Comment by akodali18** [comment]
No longer making the review changes here, need to make the same changes on the new repo.

**Comment by johncornish** [comment]
Already implemented in operator
